pages/styles/portfolio.css

```diff
@@ -0,0 +1,35 @@
+body {
+    background-color: rgb(252, 196, 93);
+}
+
+header {
+    background-color: rgb(123, 156, 72);
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif; 
+    font-style: italic;
+    font-size: 22px;
+    border-radius: 25px;
+    text-align: center;
+    padding: 30px;
+    margin: 20px;
+}
```

Comment:
There's quite a bit of overlap with the two `header` selectors in your css files. Maybe we can combine the repeated rules, and then put only what's different in `portfolio` here.